rpc/dialer.go

```diff
@@ -302,11 +309,18 @@ func dialDirectGRPC(ctx context.Context, address string, dOpts *dialOptions, log
 			dialOptsCopy := *dOpts
 			dialOptsCopy.insecure = dOpts.externalAuthInsecure
 			dialOptsCopy.externalAuthAddr = ""
+			dialOptsCopy.externalAuthMaterial = ""
 			dialOptsCopy.creds = Credentials{}
+			dialOptsCopy.authEntity = ""
+
+			// reset the tls config that is used for the external Auth Service.
+			dialOptsCopy.tlsConfig = &tls.Config{MinVersion: tls.VersionTLS12}
```

Comment:
might want to single source this so we don't miss any version details in future updates